node/modules/lp2p/pubsub.go

```diff
@@ -74,7 +74,7 @@ func GossipSub(in GossipIn) (service *pubsub.PubSub, err error) {
 	}
 
 	isBootstrapNode := in.Cfg.Bootstrapper
-	drandTopic, err := getDrandTopic(in.Dr.ChainInfoJSON)
+	drandTopic, err := getDrandTopic(in.Dr[0].Config.ChainInfoJSON)
```

Comment:
We should switch to the new topic a bit before the fork, right?

Reply:
I guess this is for setting up topic params, in which case it should probably be safe to set those for all beacons first (assuming those are different per beacon)

Reply:
Yeah, we can repeat-set them.